.github/CODEOWNERS

```diff
@@ -1,2 +1,8 @@
 * @snyk/hammerhead
 **/infrastructure/code/ @snyk/zenith
+**/infrastructure/code/sast_enabled.go @snyk/nebula
+**/infrastructure/code/sast_enabled_test.go @snyk/nebula
+**/infrastructure/code/sast_local_engine.go @snyk/nebula
+**/infrastructure/code/sast_local_engine_test.go @snyk/nebula
+**/infrastructure/code/code_test.go @snyk/nebula
+**/infrastructure/code/code.go @snyk/nebula
```

Comment:
As the last matching pattern takes precedence (see [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)), please change all nebula lines to `@snyk/zenith @snyk/nebula`, optionally `@snyk/hammerhead @snyk/zenith @snyk/nebula`

Reply:
Done